cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -51,13 +71,33 @@ def _init_options(self, kwargs):
 
     def _run_task(self):
         options = self.options['options'].copy()
+        options['SuiteNames'] = self.options['suites']
+
+        settings = RobotSettings(options)
         if 'test' in self.options:
-            options['test'] = self.options['test']
+            settings['TestNames'] = self.options['test']
         if 'include' in self.options:
-            options['include_tag'] = self.options['include']
+            settings['Include'] = self.options['include']
         if 'exclude' in self.options:
-            options['exclude_tag'] = self.options['exclude']
+            settings['Exclude'] = self.options['exclude']
+
+        # Inject CumulusCIRobotListener to build the CumulusCI library instance
+        # from self.project_config instead of reinitializing CumulusCI's config
+        listener = CumulusCIRobotListener(self.project_config, self.org_config.name)
+        settings['Listeners'] = [listener]
 
+        # Build the top level test suite
+        suite = TestSuiteBuilder().build(options['SuiteNames'])
+        suite.configure(**settings.suite_config)
+
+        # Run the test suite
+        with pyloggingconf.robot_handler_enabled(settings.log_level):
```

Comment:
can you add a comment of why we're creating the suites ourself? and link back to robot.run? 

is it because of the include/exlude? robot run should be able to take a listener.